packages/storage-plus/src/indexed_map.rs

```diff
@@ -337,6 +342,19 @@ mod test {
             .count();
         assert_eq!(0, count);
 
+        // index_key() works
+        let key = map
+            .idx
+            .name
+            .index_key((PkOwned(b"Maria".to_vec()), PkOwned(b"1".to_vec())));
```

Comment:
This is very unclear what the second item is.
Is this the primary key that she is stored under? (pk)?

If so, let's filter with only some of them (eg. all Marias with pk >= 2)

Reply:
It is the primary key, yes. Maybe a good idea would be to show an example with this is empty (`b""`). Will add a couple comments / more examples.